serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -86,7 +86,8 @@ def __init__(
 
         self.engine_config = engine_config
         self.model_artifact_config = model_artifact_config
-        self.text_generator = PagedCacheModelTextGenerator(model)
+        # TODO(@team): fix type checking issue. currently ignore to make mypy happy.
+        self.text_generator = PagedCacheModelTextGenerator(model)  # type: ignore
```

Comment:
Possibly it is question to @masahi:
`PagedCacheModelTextGenerator` expects `model` with type `TextGenerator`:
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/paged_cache_model.py#L25
But if follow to it `model` has type `Model`:
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/paged_cache_model.py#L89
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/paged_cache_model.py#L85
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/tvm_model.py#L404C1-L406C41
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/tvm_model.py#L468
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/tvm_model.py#L409
https://github.com/octoml/mlc-llm/blob/c36d47c1ec7b2d0f331f33d1795e3f7a1cfaf42d/serve/mlc_serve/model/tvm_model.py#L131

Reply:
It is the same issue as https://github.com/octoml/mlc-llm/issues/138. I don't understand what's wrong @yelite

Reply:
@masahi `Model` type was replaced by `TextGenerator` in #159. May be it is typo or there is some idea, but it was not implemented anyhow.

Reply:
@vvchernov `TextGenerator` is a protocol, and `Model` class implements it. So it is valid to pass an instance of `Model` to places which expect `TextGenerator`.

Reply:
May it is some specific of python which I do not know, but I expect for example that `Model` is inherited from `TextGenerator` It will be clearer

Reply:
@sunggg You can now remove this mypy workaround.